test/builds/browser_build_as_commonjs.js

```diff
@@ -21,4 +21,13 @@ API.forEach(n => {
   assert(_ => keys.includes(n), `API should include ${n}`);
 });
 
+// See e.g. highlightjs/highlight.js#3263
+const langs = ["python", "python-repl"]
```

Comment:
Added tests here since it is specific to the brower build.

Reply:
Yes, this is a helpful addition for sure and regression test. :)